### PR TITLE
[ELY-2446] Add a test class for RegexNameRewriter

### PR DESCRIPTION
**Pull request by mittalaishwarya**
https://issues.redhat.com/browse/ELY-2446

### CODE REVIEW

File: auth/util/src/test/java/org/wildfly/security/auth/util/RegexNameRewriterTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    private static final String BEFORE_REPLACE_USER_NAME = ".user.";
+    private static final String AFTER_REPLACE_ALL_USER_NAME = "dotuserdot";
+    private static final String AFTER_REPLACE_USER_NAME = "dotuser.";
+
+    @Test
+    public void testRegexNameRewriter() {
+        Pattern pattern = Pattern.compile("\\.");
+
+        RegexNameRewriter regexNameRewriter = new RegexNameRewriter(pattern, "dot", true);
+        MutableNameRewriter rewriter = new MutableNameRewriter(regexNameRewriter);
```
Comment (skyllarr) [inline auth/util/src/test/java/org/wildfly/security/auth/util/RegexNameRewriterTest.java]: @mittalaishwarya You don't need to use MutableNameRewriter at all to test RegexNameRewriter. You can just call method rewriteName on the regexNameRewriter

File: auth/util/src/test/java/org/wildfly/security/auth/util/RegexNameRewriterTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package org.wildfly.security.auth.util;
+
+import org.junit.Test;
+import org.wildfly.security.auth.util.RegexNameRewriter;
```
Comment (skyllarr) [inline auth/util/src/test/java/org/wildfly/security/auth/util/RegexNameRewriterTest.java]: @mittalaishwarya Just a minor,  this class is in the same package as this test. So you don't have to import it

**Comment by skyllarr** [comment]
@mittalaishwarya Otherwise this looks good!

**Comment by jessicarod7** [comment]
Hey, if you could squash these changes into one commit, that would be awesome. Thanks!

**Review by skyllarr — APPROVED** [review]

**Comment by skyllarr** [comment]
@mittalaishwarya Looks great, thank you!

**Review by fjuma — APPROVED** [review]
Sorry for the delay in getting this one merged, I've been away for a bit.

Thanks very much @mittalaishwarya!

**Pull request merged**